spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/DefaultRowWriter.scala

```diff
@@ -9,9 +9,15 @@ import com.datastax.spark.connector.types.TypeConverter
 import scala.collection.{Map, Seq}
 import scala.collection.JavaConversions._
 
+trait CheckSetting
+object CheckLevel{
```

Comment:
So i can use the RowWriter factory without it complaining that i'm missing keys to get values to put in the driver